src/Rector/Contrib/PHPUnit/SpecificMethodInternalTypeRector.php

```diff
@@ -0,0 +1,102 @@
+<?php declare(strict_types=1);
+
+namespace Rector\Rector\Contrib\PHPUnit;
+
+use PhpParser\Node;
+use PhpParser\Node\Arg;
+use PhpParser\Node\Expr\FuncCall;
+use PhpParser\Node\Expr\MethodCall;
+use PhpParser\Node\Identifier;
+use PhpParser\Node\Scalar\String_;
+use Rector\NodeAnalyzer\MethodCallAnalyzer;
+use Rector\Rector\AbstractRector;
+
+/**
+ * Before:
+ * - $this->assertTrue(is_{internal_type}($anything));
+ * - $this->assertFalse(is_{internal_type}($anything));
```

Comment:
@TomasVotruba Is this description acceptable?

Reply:
Yep, all clear